### DISCUSSION TITLE
Cannot create project because of old repo/ssh key

### DISCUSSION THREAD

**Discussion by CodesByMo** [community/community#22467] [New to GitHub]
<p>I keep getting this error</p>
<pre><code class="lang-auto">npm ERR! code 128
npm ERR! An unknown git error occurred
npm ERR! command git --no-replace-objects ls-remote ssh://git@github.com/[InternshipCompanyName]/stream-browser.git
npm ERR! ERROR: Repository not found.
npm ERR! fatal: Could not read from remote repository.
npm ERR!
npm ERR! Please make sure you have the correct access rights
npm ERR! and the repository exists
</code></pre>
<p>The repo in question was from an internship from more than 2 years ago, I haven’t touched any code since then and I just wanted to start a new project to brush the dust off but I’ve been stuck now for more than 6 hours searching every corner of google &amp; stackoverflow for some semblance of an answer. I’ve tried absolutely everything and I cannot create a project/install any packages at all anywhere on my hard drive(s). I have installed Node, npm, git, github desktop and uninstalled and re-installed them about 25 times. I’ve re-authenticated an added a new SSH key to my Github account. I do not know what else to do, I have no access to this repo and I believe it was most likely deleted from the Organization’s account.</p>
<p>Is it possible to simply remove this from my account completely? What can I do?</p>

**Comment by jsoref** [comment]
<p>While logged, can you visit:</p>
<pre><code class="lang-auto">https://github.com/[InternshipCompanyName]/stream-browser/
</code></pre>
<p>If not, then they’ve either deleted/renamed the repository or removed your access to it/the organization.</p>

**Comment by CodesByMo** [comment]
<p>I reached out to the owner and they confirmed it was not deleted. Is there a way I can remove myself from the repo without them?</p>

**Comment by jsoref** [comment]
<p>What do you mean you keep getting this error?</p>
<p>Is something sending you an email?</p>
<p>If you aren’t interested in a repository, you generally can not try to clone it. But if something is emailing you a thing, then you should be able to adjust whatever that is.</p>
<p>If you’re a member of the organization, you should be able to leave the organization:</p>
<aside class="onebox allowlistedgeneric" data-onebox-src="https://docs.github.com/en/github-ae@latest/account-and-profile/setting-up-and-managing-your-github-user-account/managing-your-membership-in-organizations/removing-yourself-from-an-organization">
  <header class="source">
      <img src="https://docs.github.com/assets/cb-803/images/site/favicon.svg" class="site-icon" width="16" height="16">

      <a href="https://docs.github.com/en/github-ae@latest/account-and-profile/setting-up-and-managing-your-github-user-account/managing-your-membership-in-organizations/removing-yourself-from-an-organization" target="_blank" rel="noopener">GitHub Docs</a>
  </header>

  <article class="onebox-body">
    <img src="https://user-images.githubusercontent.com/2119212/181071274-22c29a16-d192-4adf-b991-02526b327e54.png" class="thumbnail onebox-avatar" width="500" height="500" srcset="https://aws1.discourse-cdn.com/github/optimized/2X/3/337b7cc06184984e148c075df20ef009e0e6ecaa_2_500x500.png, https://aws1.discourse-cdn.com/github/optimized/2X/3/337b7cc06184984e148c075df20ef009e0e6ecaa_2_750x750.png 1.5x, https://aws1.discourse-cdn.com/github/optimized/2X/3/337b7cc06184984e148c075df20ef009e0e6ecaa_2_1000x1000.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/2X/3/337b7cc06184984e148c075df20ef009e0e6ecaa_2_10x10.png">

<h3><a href="https://docs.github.com/en/github-ae@latest/account-and-profile/setting-up-and-managing-your-github-user-account/managing-your-membership-in-organizations/removing-yourself-from-an-organization" target="_blank" rel="noopener">Removing yourself from an organization - GitHub Docs</a></h3>

  <p>If you're an outside collaborator or a member of an organization, you can leave the organization at any time.</p>


  </article>

  <div class="onebox-metadata">
    
    
  </div>

  <div style="clear: both"></div>
</aside>

<p>If you’re a collaborator on a repository, you should be able to leave that as well:</p>
<aside class="onebox allowlistedgeneric" data-onebox-src="https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-github-user-account/managing-access-to-your-personal-repositories/removing-yourself-from-a-collaborators-repository">
  <header class="source">
      <img src="https://docs.github.com/assets/cb-803/images/site/favicon.svg" class="site-icon" width="16" height="16">

      <a href="https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-github-user-account/managing-access-to-your-personal-repositories/removing-yourself-from-a-collaborators-repository" target="_blank" rel="noopener">GitHub Docs</a>
  </header>

  <article class="onebox-body">
    <img src="https://user-images.githubusercontent.com/2119212/181071275-722b13d7-76f0-4763-a216-006bc64604ee.png" class="thumbnail onebox-avatar" width="500" height="500" srcset="https://aws1.discourse-cdn.com/github/optimized/2X/3/337b7cc06184984e148c075df20ef009e0e6ecaa_2_500x500.png, https://aws1.discourse-cdn.com/github/optimized/2X/3/337b7cc06184984e148c075df20ef009e0e6ecaa_2_750x750.png 1.5x, https://aws1.discourse-cdn.com/github/optimized/2X/3/337b7cc06184984e148c075df20ef009e0e6ecaa_2_1000x1000.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/2X/3/337b7cc06184984e148c075df20ef009e0e6ecaa_2_10x10.png">

<h3><a href="https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-github-user-account/managing-access-to-your-personal-repositories/removing-yourself-from-a-collaborators-repository" target="_blank" rel="noopener">Removing yourself from a collaborator's repository - GitHub Docs</a></h3>

  <p>If you no longer want to be a collaborator on someone else's repository, you can remove yourself.</p>


  </article>

  <div class="onebox-metadata">
    
    
  </div>

  <div style="clear: both"></div>
</aside>

<p>But, it’s really unclear what’s going on here.</p>
<p>You haven’t really answered the question I asked in:</p>
<aside class="quote" data-post="2" data-topic="234893">
  <div class="title">
    <div class="quote-controls"></div>
    
    <a href="https://github.community/t/cannot-create-project-because-of-old-repo-ssh-key/234893/2">Cannot create project because of old repo/ssh key</a> <a class="badge-wrapper  bullet" href="/c/github-help/48"><span class="badge-category-bg" style="background-color: #6a737d;"></span><span style="" data-drop-close="true" class="badge-category clear-badge" title="Need help with your GitHub account or the core features of GitHub? This is the place to look for answers.">GitHub Help</span></a>
  </div>
  <blockquote>
    While logged, can you visit: 
https://github.com/[InternshipCompanyName]/stream-browser/

If not, then they’ve either deleted/renamed the repository or removed your access to it/the organization.
  </blockquote>
</aside>

**Comment by CodesByMo** [comment]
<p>I have checked my github account, I am no longer apart of the Organization, I am not a collaborator on the specified repo giving me the error <code>npm ERR! command git --no-replace-objects ls-remote ssh://git@github.com/[InternshipCompanyName]/stream-browser.git</code></p>
<p>I reinstalled my Windows a few months ago and hadn’t touched any of the repos in 2 years prior to that. I also installed Git &amp; Node earlier for the first time since the fresh Windows install.</p>
<p>Also, I cannot access <code>https://github.com/[InternshipCompanyName]/stream-browser/</code><br>
It is a 404 Error page</p>

**Comment by jsoref** [comment]
<p>Ok, so you can’t access it, and you’re no longer a member/collaborator, that makes sense.</p>
<p>What is running this <code>npm</code> command? Is it a program on your computer? Or a job?</p>
<p>It seems likely you have a <code>package.json</code> file somewhere that references <code>ssh://git@github.com/[InternshipCompanyName]/stream-browser.git</code>.</p>

**Comment by CodesByMo** [comment]
<p>I’m running the npm command in a new folder (via VSCode, but I’ve tried using CMD &amp; Git) to install a package for the project I’m working on. I’ve Windows searched for <code>ssh://git@github.com/[InternshipCompanyName]/stream-browser.git</code> on my main drive and it doesn’t return any results. I’m unsure how I would go about locating it otherwise.</p>

**Comment by jsoref** [comment, marked as answer]
<p>Look for a <code>.npm</code> directory in <code>%HOMEDRIVE%%HOMEPATH%</code> (also check <code>.npmrc</code>).</p>
<p>Personally, I’d use:</p>
<aside class="onebox allowlistedgeneric" data-onebox-src="https://docs.microsoft.com/en-us/sysinternals/downloads/procmon">
  <header class="source">

      <a href="https://docs.microsoft.com/en-us/sysinternals/downloads/procmon" target="_blank" rel="noopener">docs.microsoft.com</a>
  </header>

  <article class="onebox-body">
    <img src="https://user-images.githubusercontent.com/2119212/181071278-a316a019-fdfc-4b7e-a823-492879ee79da.png" class="thumbnail onebox-avatar" width="400" height="400">

<h3><a href="https://docs.microsoft.com/en-us/sysinternals/downloads/procmon" target="_blank" rel="noopener">Process Monitor - Windows Sysinternals</a></h3>

  <p>Monitor file system, Registry, process, thread and DLL activity in real-time.</p>


  </article>

  <div class="onebox-metadata">
    
    
  </div>

  <div style="clear: both"></div>
</aside>

<p>And have it log <code>npm</code> or <code>node</code> or something. w/ a bit of filtering, it’d eventually be pretty obvious which file is contributing to my pain.</p>

**Comment by CodesByMo** [comment]
<p>THANK YOU! I found the package-lock.json pretty deep in my files on my C: drive</p>
<p>I’m successfully able to install the npm package to my project, I really appreciate the help.</p>
